api/cpp/tests/multiple-includes/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# Copyright © SixtyFPS GmbH <info@slint-ui.com>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+add_executable(multiple-includes main.cpp logic.cpp)
+target_link_libraries(multiple-includes PRIVATE Slint::Slint)
+slint_target_sources(multiple-includes appwindow.slint)
```

Comment:
Ideally this would be a separate CMake build driven by CTest like in corrosion, so that this executable isn't floating around (in bin). But that's not in scope of this change and only happens when building with tests enabled anyway.